lib/resolver/live-resolver-query.js

```diff
@@ -1,4 +1,9 @@
 export default function ({ type, target }) {
+  if (type == 'npm') {
+    // If the target looks like 'foo/bar.js', pretend it is 'foo' instead. See
+    // https://github.com/OctoLinker/browser-extension/issues/93
+    target = target.split('/')[0];
+  }
```

Comment:
I think, the `liveResolverQuery` shouldn't know anything about npm or something else. He should care about building the live resolver request url based on the passed args. Imagine we want to have the same behaviour for another languages lets say PHP. In that case we would have to pollute the `liveResolverQuery` with logic for handling PHP. That's my concern, why it shouldn't be in here.